frontend/src/layouts/userLayout.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Head from 'next/head'
+
+import Header from '~/components/user/Header'
+import Footer from '~/components/admin/Footer'
+import { classNames } from '~/utils/classNames'
+
+type Props = {
+  children: React.ReactNode
```

Comment:
```suggestion
  children: ReactNode
```

import RectNode nalang